{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://bitbucket.org/pytest-dev/pytest-timeout",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-timeout/pytest-timeout-1.0.0.tar.gz",
    "subdirs": [
      "osx-64"
    ],
    "summary": "This is a plugin which will terminate tests after a certain timeout.",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.0.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003",
    "cf201901"
  ],
  "package": "pytest-timeout-1.0.0-py34_0.tar.bz2",
  "repodata": {
    "build": "py34_0",
    "build_number": 0,
    "depends": [
      "pytest >=2.8.0",
      "python 3.4*"
    ],
    "license": "MIT",
    "md5": "70c1d6c70485bb247a9d2d83841fd612",
    "name": "pytest-timeout",
    "sha256": "a66dd36231363fe234452c4a41feef9de7289ab3160d8926f90e7e127ab5c9c7",
    "size": 11468,
    "subdir": "osx-64",
    "version": "1.0.0"
  },
  "repodata_version": 1,
  "subdir": "osx-64",
  "url": "https://conda.anaconda.org/conda-forge/osx-64/pytest-timeout-1.0.0-py34_0.tar.bz2"
}